examples/4337-passkeys/src/components/SendNativeToken.tsx

```diff
@@ -0,0 +1,162 @@
+    feeData?.maxPriorityFeePerGas != null
+
+  const userOperationFee = useMemo(() => {
+    if (!gasParametersReady) {
+      return 0n
+    }
+
+    // @ts-expect-error it is handled in the if statement above
+    return getMissingAccountFunds(feeData!.maxFeePerGas, userOpGasLimitEstimation!, 0n)
+  }, [feeData, gasParametersReady, userOpGasLimitEstimation])
+  const missingAccountFundsForEntrypoint =
+    userOperationFee - accountEntryPointBalance > 0 ? userOperationFee - accountEntryPointBalance : 0n
+  const missingAccountFundsFromTheBalance = balanceWei - missingAccountFundsForEntrypoint
+  const maxAmount = missingAccountFundsFromTheBalance >= 0n ? ethers.formatEther(balanceWei - missingAccountFundsForEntrypoint) : '0'
+
+  const send: FormEventHandler<HTMLFormElement> = async (e) => {
+    e.preventDefault()
+    setError('')
+    setUserOpHash(null)
+
+    if (!ethers.isAddress(to)) {
+      setError('Invalid address')
+      return
+    }
+
+    if (amount === null || ethers.parseEther(amount) > balanceWei || missingAccountFundsFromTheBalance < 0n) {
+      setError('Invalid amount')
+      return
+    }
+
+    if (!gasParametersReady) return
+
+    const userOpToSign: UnsignedPackedUserOperation = {
+      ...userOp,
+      ...packGasParameters({
+        verificationGasLimit: userOpGasLimitEstimation.verificationGasLimit,
+        callGasLimit: userOpGasLimitEstimation.callGasLimit,
+        maxPriorityFeePerGas: feeData?.maxPriorityFeePerGas,
+        maxFeePerGas: feeData?.maxFeePerGas,
+      }),
+      preVerificationGas: userOpGasLimitEstimation.preVerificationGas,
+    }
+    try {
+      const userOpHash = await onSend(userOpToSign)
+      setUserOpHash(userOpHash)
+    } catch (e) {
+      console.error(e)
+      setError('Error sending transaction')
+    }
+  }
+
+  return (
+    <form
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '1rem',
+      }}
+      onSubmit={send}
+    >
+      <h2>Send Native Token</h2>
+      <label>
+        To:
+        <input required type="text" name="to" value={to} onChange={(e) => setTo(e.target.value)} style={{ display: 'block' }} />
+      </label>
+      <label>
+        Amount (ETH):
+        <input
+          required
+          type="number"
+          name="amount"
+          value={amount}
+          onChange={(e) => setAmount(e.target.value)}
+          style={{ display: 'block' }}
+        />
+        <button
+          type="button"
+          style={{
+            padding: 0,
+            background: 'transparent',
+            fontSize: '0.8rem',
+          }}
+          onClick={() => setAmount(maxAmount)}
+        >
+          Max
+        </button>
+      </label>
+      <p>Approx. fee: {ethers.formatEther(userOperationFee)} ETH</p>
+      {missingAccountFundsFromTheBalance < 0n && (
+        <MissingAccountFundsCard
+          provider={walletProvider}
+          missingAccountFunds={abs(missingAccountFundsFromTheBalance)}
```

Comment:
Shouldn't this just be:

```suggestion
          missingAccountFunds={-missingAccountFundsFromTheBalance}
```

Because of the: `      {missingAccountFundsFromTheBalance < 0n && (`

Reply:
Somehow I forgot that this is supported in programming languages 🤦